packages/qwik/src/qwikloader.ts

```diff
@@ -52,7 +52,17 @@ export const qwikLoader = (doc: Document, hasInitialized?: number) => {
   const dispatch = async (element: Element, onPrefix: string, ev: Event, eventName = ev.type) => {
     const attrName = 'on' + onPrefix + ':' + eventName;
     if (element.hasAttribute('preventdefault:' + eventName)) {
-      ev.preventDefault();
+      if (
+        !element.hasAttribute('allowmodifiers') ||
+        !(
+          (ev as MouseEvent).ctrlKey ||
+          (ev as MouseEvent).metaKey ||
+          (ev as MouseEvent).shiftKey ||
+          (ev as MouseEvent).altKey
+        )
+      ) {
+        ev.preventDefault();
+      }
```

Comment:
You always deliver @jordanw66. This dispatch is for all events right?

So this might be a good check and shortens the code a bit.
```suggestion
      if (
        !element.hasAttribute('allowmodifiers') ||
        !(ev instanceof MouseEvent && (ev.ctrlKey || ev.metaKey || ev.shiftKey || ev.altKey))
      ) {
        ev.preventDefault();
```

I'm maybe thinking to generic here and you might be right, this is enough.
As of right now, I have no clue how to document this that everyone gets it. Maybe this is not necessary as react-router-link works in the same way I guess.

Reply:
@jordanw66 valid point, but what happens on keyup?

Speaking of performance, instanceof will be faster that the suggestion here: #4496.   
The alternative would be casting to unknown and then check with ```in```.

I guess you abort this one and create a new one? Let me know